.travis.yml

```diff
@@ -13,10 +13,12 @@ cache:
   yarn: true
 
 install:
+  - gem install rubocop
```

Comment:
Yes, it should be in the Gemfile. If it is a development dependency, it should be there.